File: pset1/cash/cash.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>

int main(void)
{
    float change;

    do
    {
        change = get_float("Change owed:"); // Gets change that should be returned
    }
    while (change < 0); // Loops to check if its not negative

    //25, 10, 5, 1

    int cents = round(change * 100);

    int quarters = cents / 25;
    int dimes = (cents % 25) / 10;
    int nickels = ((cents % 25) % 10) / 5;
    int pennies = ((cents % 25) % 10) % 5;
    int total_coins = quarters + dimes + nickels + pennies;

    printf("%i\n", total_coins);
}

File: pset6/cash/cash.py
from cs50 import get_float

while True:
    # gets change to be calculated and checks if its higher than 0
    change = get_float("Change owed:")
    if change > 0:
        # rounds it ups so it can be treated
        cents = round(change * 100)

        # uses module division as per pset1 in C
        quarters = cents // 25
        dimes = (cents % 25) // 10
        nickels = ((cents % 25) % 10) // 5
        pennies = ((cents % 25) % 10) % 5
        coins = quarters + dimes + nickels + pennies

        print(coins)
        break